community/pike/APKBUILD

```diff
@@ -0,0 +1,187 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="GPL-2.0-or-later"
+options="!check" # there is no tests for those packages
+depends=
+makedepends="gmp-dev
+	bzip2-dev
+	freeglut-dev
+	gdbm-dev
+	giflib-dev
+	giflib-utils
+	gtk+2.0
+	libglade-dev
+	libsm-dev
+	mesa-gl
+	nettle
+	nettle-dev
+	nettle-utils
+	pcre-dev
+	sdl-dev
+	sdl_mixer-dev
+	sqlite-dev
+	sqlite-libs
+	readline
+	zlib-dev
+	librsvg-dev"
+install=
+subpackages="$pkgname-doc $pkgname-full"
+source="http://pike.lysator.liu.se/pub/pike/all/${pkgver}/Pike-v${pkgver}.tar.gz"
+builddir="$srcdir/Pike-v$pkgver"
+
+_full_deps="$pkgname=${pkgver}-r${pkgrel} ${pkgname}-doc=${pkgver}-r${pkgrel}"
+for _name in bzip2 gdbm gl gtk image pcre sdl svg sqlite
+do
+	subpackages="$subpackages $pkgname-${_name}"
+	_full_deps="${_full_deps} $pkgname-${_name}=$pkgver-r$pkgrel"
+done
```

Comment:
Consider adding `$pkgname-dev` to the end of `subpackages` to split out the include files

Reply:
There are at least two headers, that would have to be moved back manually to main package. Pike's preprocesor needs them while parsing some core files.